### DISCUSSION TITLE
Make the increased speed adjustable

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#49] [Feedback]
In addition to this post (https://github.com/Mojang/minecraft-editor/discussions/27 ) about how it's a struggle to go to a certain position;

Make the increased speed adjustable.

The increased speed could be customly amplifier of x times normal speed. In general, setting a higher custom travelling speed in bedrock would be quite handy for creators on and off the editor.

**Comment by minecraftchloe** [comment]
This would be amazing, I completely agree! The trick is figuring out the optimal way to do it where Selection Mode -> Fixed Distance (using the scrollwheel to adjust distance from you) is still usable and there aren't conflicts.
